task/views.py

```diff
@@ -0,0 +1,84 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render, redirect, get_object_or_404
+from django.urls import reverse_lazy
+from django.views import generic
+
+from task.forms import TaskCreateForm, TagCreateForm
+from task.models import Task, Tag
+
+
+class TaskListView(generic.ListView):
+    model = Task
+    paginate_by = 4
+    template_name = "task/index.html"
+
+    def get_queryset(self):
+        queryset = Task.objects.order_by("task_done")
+        return queryset
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(TaskListView, self).get_context_data(**kwargs)
+        context["list_tasks"] = context["object_list"]
+
+        return context
+
+class TagListView(generic.ListView):
+    model = Tag
+    paginate_by = 4
+    template_name = "task/tag.html"
```

Comment:
You don't have such template
tags.html I suppose

Reply:
edited